scripts/setup-ubuntu.sh

```diff
@@ -86,6 +86,31 @@ function install_velox_deps {
   run_and_time install_folly
 }
 
+function install_gcs_deps {
```

Comment:
Is this used anywhere? I guess you added `setup-gcs.sh` instead.

Reply:
removed